src/uu/factor/Cargo.toml

```diff
@@ -14,11 +14,19 @@ edition = "2018"
 [lib]
 path = "src/factor.rs"
 
+[build-dependencies]
+num-traits="0.2"
```

Comment:
are you sure it needs to be duplicated?
if there is a good reason, please document it :)

Reply:
I added a documentation comment, thanks  <3